### PR TITLE
Bunch of copy changes

### PR DESCRIPTION
**Pull request by paulsmith**
My goal is to update the Ad Hoc site to:

- Make it crystal-clear what we actually do (i.e., what value we are delivering to our govt customers)
- Tell the origin story and put it up-front -- we rescued HealthCare.gov, and Ad Hoc came out of that -- that resonates with people
- Associate ourselves with the larger reform world - USDS, etc.
- Remove vagueness or jargon
- Make a more compelling and inclusive pitch to potential recruits, including stating (briefly) company policies, benefits, etc.
- Tell more success stories -- flesh out vets.gov (even if more is coming on this later), Window Shopping (now that OE 2016 is over)
- Say what makes us unique - private sector, yeah, but also, we're smart and experienced, can co-create with policy experts a product, avoiding the far riskier requirements cycle

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,13 +8,29 @@
...
 </div>
 
+<div class="pitch about">
+  <p>
+    Ad Hoc is a small software company that came out of the successful
+    HealthCare.gov rescue effort.
+  </p>
+  <p>
+    We design, build, and operate service-delivering software that's fast,
+    stable, efficient, and usable.
```
Comment (greggersh) [inline index.html]: I think we should say that we design, build, and operate services for citizens (the fact that they are software is incidental).

File: index.html [hunk 2]
```diff
@@ -8,13 +8,29 @@
...
+  </p>
+  <p>
+    <a href="/about/">Find out more about Ad Hoc and start realizing
+      your policy objectives with software that actually works.</a>
```
Comment (greggersh) [inline index.html]: I think policy objectives is ok for now, but I feel like there's a better way to say this.

File: pages/about-healthcare.gov-rescue.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          could enroll hundreds of thousands of Americans each day in affordable
+          health coverage, and the organization and processes it established
+          during the rescue put it well on its way to becoming a stable part of
+          the American welfare system.
+        </p>
+
+        <p>A fuller retelling of the HealthCare.gov can be read in the TIME
+          magazine cover story by Steven Brill,
+          "<a href="http://time.com/10228/obamas-trauma-team/">Obama's Trauma
+          Team</a>".
```
Comment (greggersh) [inline pages/about-healthcare.gov-rescue.html]: We should put a picture of the time cover, maybe link to chris hayes, and the organizing for america speeches.

File: pages/about.html [hunk 4]
```diff
@@ -1,14 +1,80 @@
...
+        <h2>About Ad Hoc</h2>
+        <p>
+          Ad Hoc is a small software engineering company that came out of the
+          successful effort to rescue HealthCare.gov after its disastrous
+          initial launch.
+        </p>
+
+        <p>
+          Ad Hoc is focused on delivering and operating fast, stable, and
+          well-designed software on behalf of U.S. federal and state government
```
Comment (greggersh) [inline pages/about.html]: replace software with services

File: pages/about.html [hunk 5]
```diff
@@ -1,14 +1,80 @@
...
+          that enables policy-makers to provide transactional and informational
```
Comment (greggersh) [inline pages/about.html]: replace policy-makers with agencies.

File: pages/about.html [hunk 6]
```diff
@@ -1,14 +1,80 @@
...
+          services to Americans. We do this by bringing people from startups and
```
Comment (greggersh) [inline pages/about.html]: to the American people.

File: pages/about.html [hunk 7]
```diff
@@ -1,14 +1,80 @@
...
+          The <a href="https://playbook.cio.gov/">U.S. Digital Services
+          Playbook</a>, and
+          the <a href="https://www.whitehouse.gov/digital/united-states-digital-service">United
+          States Digital Service (U.S.D.S.)</a>, came directly out of the
+          HealthCare.gov rescue, with a goal of institutionalizing what the "ad
+          hoc team" did that worked so successfully.
+        </p>
+
+        <p>
+          Ad Hoc LLC complements the U.S.D.S, along
```
Comment (greggersh) [inline pages/about.html]: Not an abbreviation, usually.

File: pages/about-healthcare.gov-rescue.html (hunk 3 above)
Comment (greggersh) [inline pages/about-healthcare.gov-rescue.html]: Oh and the wired article to prove I exist.

File: index.html (hunk 1 above)
Comment (paulsmith) [inline index.html]: Sort of -- we don't do anything *but* make software so it's a little strange to me to leave that out. I don't feel strongly about it though.

Re: "citizens" -- not everything is just for citizens, right? Lots of government services are for aliens, residents, even undocumented immigrants are entitled to certain things. It's tricky but we should probably avoid "citizens" or "Americans" even.

File: index.html (hunk 2 above)
Comment (paulsmith) [inline index.html]: Goals?

File: pages/about-healthcare.gov-rescue.html (hunk 3 above)
Comment (paulsmith) [inline pages/about-healthcare.gov-rescue.html]: Yep

File: index.html (hunk 1 above)
Comment (paulsmith) [inline index.html]: Maybe "American public" or just "consumers"

**Comment by paulsmith** [comment]
@greggersh I think this is in good shape for now, but please review and see what you think.

**Comment by greggersh** [comment]
Looks good! :+1:

**Comment by greggersh** [comment]
Will need to rebase to get the project manager job added to the jobs page.

**Comment by paulsmith** [comment]
Will do.

**Pull request closed without merging**